tests/safe_test.ts

```diff
@@ -197,6 +197,22 @@ Clarinet.test({
         ]);
         assertEquals(block.receipts[0].result.expectOk(), "true");
 
+        // The transaction already confirmed. Should revert
+        block = CHAIN.mineBlock([
+            Tx.contractCall(
+                "safe",
+                "confirm",
+                [
+                    types.uint(0), 
+                    types.principal("ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.add-owner"),
+                    types.principal("ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.safe")
+                ],
+                WALLETS[2]
+              ),
+        ]);
+        assertEquals(block.receipts[0].result.expectErr(), "u180");
```

Comment:
I would advise to always check full type. By comparing result only to string, you risk that at some point you'll return different type than you intended and you won't catch it.

You can do it as follows:
```ts
block.receipts[0].result.expectErr().expectUint(180);
```
or for better readability:
```ts
const ERR_TX_CONFIRMED = 180;
block.receipts[0].result.expectErr().expectUint(ERR_TX_CONFIRMED);
```